### PR TITLE
Multiply Pneumonia dataset

### PR DESCRIPTION
**Pull request by garg-amit**
## Purpose

* Provide a pipeline to multiply the Pneumonia dataset.
* Add test to CI/CD.

## What is the expected review turnaround time?

Urgency:
- [ ] High (needs review today)
- [x] Medium (needs review within a few days - most common)
- [ ] Low (can wait a week)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

Run the Pneumonia upload data pipeline followed by the multiply data pipeline.

## What to Check
Verify that the following are valid
* Look for a folder named `multiplied_data` in silos' storage accounts and verify if the files are replicated or not.

### CODE REVIEW

File: examples/pipelines/utils/multiply_data/submit.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    "--submit",
+    default=False,
+    action="store_true",
+    help="actually submits the experiment to AzureML",
+)
+
+parser.add_argument(
+    "--example",
+    required=True,
+    choices=["PNEUMONIA"],
```
Comment (jfomhover) [inline examples/pipelines/utils/multiply_data/submit.py:52]: you don't actually need this parameter, we can use this pipeline to multiply any dataset with files.

**Review by jfomhover — CHANGES_REQUESTED** [review]

File: examples/components/PNEUMONIA/multiply_data/run.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+"""This script multiplies a folder by copying the files and adding suffixes."""
```
[the remark this answers is not in the document]
Comment (garg-amit) [reply examples/components/PNEUMONIA/multiply_data/run.py]: Moved and renamed it to `multiple_data_files`

File: examples/pipelines/utils/multiply_data/submit.py (hunk 1 above)
Comment (garg-amit) [reply examples/pipelines/utils/multiply_data/submit.py]: Removed!

**Review by majercakdavid — COMMENTED** [review]
Looks great! Maybe a small documentation on how to use it would be useful.

**Review by majercakdavid — APPROVED** [review]

**Review by jfomhover — APPROVED** [review]

**Pull request merged**